From 11ec6a5bb6978f379206de5f2df9fc7c79a9a1c8 Mon Sep 17 00:00:00 2001
From: German Parente <gparente@redhat.com>
Date: Tue, 6 Nov 2018 18:46:42 +0100
Subject: [PATCH] Ticket 50013 - Log warn instead of ERR when aci target does
 not exist.

Bug Description:

This is something we have very often in IPA context and customers are very often asking why there are errors in the logs:

[31/Oct/2018:05:52:23.436616394 -0400] - ERR - NSACLPlugin - acl_parse - The ACL target cn=groups,cn=compat,dc=cgparente,dc=local does not exist
[31/Oct/2018:05:52:23.438951763 -0400] - ERR - NSACLPlugin - acl_parse - The ACL target cn=computers,cn=compat,dc=cgparente,dc=local does not exist

Fix Description:

just log WARN instead of ERR

https://pagure.io/389-ds-base/issue/50013

Author: German Parente <gparente@redhat.com>

Review by: ???
---
 ldap/servers/plugins/acl/aclparse.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/acl/aclparse.c b/ldap/servers/plugins/acl/aclparse.c
index a425adbd8..906c1d099 100644
--- a/ldap/servers/plugins/acl/aclparse.c
+++ b/ldap/servers/plugins/acl/aclparse.c
@@ -147,7 +147,7 @@ acl_parse(Slapi_PBlock *pb, char *str, aci_t *aci_item, char **errbuf)
                     slapi_search_internal_pb(temppb);
                     slapi_pblock_get(temppb, SLAPI_PLUGIN_INTOP_RESULT, &rc);
                     if (rc != LDAP_SUCCESS) {
-                        slapi_log_err(SLAPI_LOG_ERR, plugin_name,
+                        slapi_log_err(SLAPI_LOG_WARNING, plugin_name,
                                       "acl_parse - The ACL target %s does not exist\n", slapi_sdn_get_dn(&targdn));
                     }
 
-- 
2.17.2

